src/main/java/com/eliotlash/molang/variables/ExecutionContext.java

```diff
@@ -59,6 +61,9 @@ public ExecutionContext(Evaluator evaluator) {
         registerFunction("math", new Lerp("lerp"));
         registerFunction("math", new LerpRotate("lerprotate"));
         registerFunction("math", new Random("random"));
+
+        registerFunction("system", new Print("print"));
+        registerFunction("strings", new StrEquals("equals"));
```

Comment:
Let's do `string.equals` for consistency

Reply:
Changed to string equals & added a string.equalsIgnoreCase